packages/cssnano-preset-default/src/__tests__/integrations/basscss5.css

```diff
@@ -18,13 +18,13 @@ th{font-weight:700;text-align:left}
 td,th{line-height:inherit;padding:.25rem 1rem}
 th{vertical-align:bottom}
 td{vertical-align:top}
-body{font-size:100%;line-height:1.5}
+body{line-height:1.5;font-size:100%}
```

Comment:
Some strange reordering of declarations in all integration tests. Shouldn't these be grouped alphabetically?

Reply:
No, since `font-size` could conflict with `font` it is not being moved.

Reply:
What do you mean? There' s no `font` declaration here.

Reply:
It doesn't look at a selector by selector basis, the way this PR makes it safer is by not sorting properties that could conflict at all.

Reply:
I really don't know what you mean, and there's no tests to infer meaning from either. Can you expand on what you're saying?

Reply:
Ah sorry I missed this reply.

I'll add an extra test, what I mean is that any property that has a conflicting property will not be sorted.

Reply:
I don't understand what you mean by conflicting. Conflicting with *what*? The functionality, as it stands, is completely opaque and non-intuitive. The tests aren't communicating its behaviour, and that means we can't have any confidence in its result.

Reply:
:thinking: Ah I see, it completely makes sense to me but I know how the module works internally. Conflicting as in the value the property changes can also be changed by other properties, I've added tests to clarify the internal behavior. Could you check them to see if it clears the doubts up?

Reply:
Clarified this behaviour with a test: `move properties that are not in safe-order.json to the start and sort remaining`. Properties that are not 'safe', in other words: properties that have a shorthand/longhand are moved to the start. What I meant with conflicting is for a value that can be changed by multiple properties such as the `font-size` in this example because it can also be changed with `font`. These properties 'conflict', maybe it's a bad word to describe this, I'm not a native speaker. :)